src/EditLabelDialog/EditLabelDialog.jsx

```diff
@@ -24,7 +24,7 @@ export default function EditLabelDialog({
   onNew = () => {},
   onEdit = () => {},
   onClose = () => {},
-  labelDialogTitle = "Edit Label",
+  labelDialogTitle = "Add Label",
   label = { _id: "", color: "#005FA8", description: "", name: "" },
```

Comment:
It's weird for a component to have a default label in this context. Shouldn't label be a **required** property?

Reply:
I think it should be